ui/pages/swaps/searchable-item-list/list-item-search/list-item-search.component.js

```diff
@@ -18,17 +20,47 @@ export default function ListItemSearch({
   fuseSearchKeys,
   searchPlaceholderText,
   defaultToAll,
+  listContainerClassName,
 }) {
   const fuseRef = useRef();
   const [searchQuery, setSearchQuery] = useState('');
+  const isTokenToDropdown =
+    listContainerClassName === 'build-quote__open-to-dropdown';
```

Comment:
It looks like `listContainerClassName` is only used as a check for whether the search should include a search for imported tokens. Instead of basing this logic on the classname of the parent, could we pass a boolean whose name makes the use of the property clear (`shouldSearchForImports` or something like that)